welkin/models/chat.py

```diff
@@ -0,0 +1,70 @@
+from welkin.models.base import Collection, Resource
+from welkin.pagination import MetaIterator
+
+
+class Chat(Resource):
+    def create(self):
+        return super().post(
+            f"{self._client.instance}/patients/{self._parent.id}/chat/inbound"
+        )
+
+    def __str__(self):
+        return f"{self.sender['clientType']} {self.message}"
+
+
+class Chats(Collection):
+    resource = Chat
+    iterator = MetaIterator
+
+    def get(
+        self,
+        pageSize: int = 20,
+        includeArchived: bool = False,
+        pageToken: str = None,
+    ):
+        params = {
+            "includeArchived": includeArchived,
+        }
+        # pageToken and pageSize are mutually exclusive
+        if pageToken:
+            params["pageToken"] = pageToken
```

Comment:
Pagination parameters should be handled by the underlying paginator class. A user should be able to pass `paginate=True` and `size=int` to any paginated collection get method, those would pass to the request method through args/kwargs, and that in turn passes to the paginator class.